tests/test_runner/test_hooks.py

```diff
@@ -931,6 +932,23 @@ def test_neptune_hook():
     hook.run.stop.assert_called_with()
 
 
+def test_dvclive_hook(tmp_path):
+    sys.modules['dvclive'] = MagicMock()
+    runner = _build_demo_runner()
+
+    (tmp_path / 'dvclive').mkdir()
+    hook = DvcliveLoggerHook(str(tmp_path / 'dvclive'))
+    loader = DataLoader(torch.ones((5, 2)))
+
+    runner.register_hook(hook)
+    runner.run([loader, loader], [('train', 1), ('val', 1)])
+    shutil.rmtree(runner.work_dir)
```

Comment:
Does we need remove `runner.work_dir`

Reply:
I need to check if it is really necessary

Reply:
It is actually needed because we are using `mkdtemp`:

https://github.com/open-mmlab/mmcv/blob/master/tests/test_runner/test_hooks.py#L967

And as stated in the official docs (https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp):

> The user of mkdtemp() is responsible for deleting the temporary directory and its contents when done with it.

We should probably update that function to use pytest tmp_path fixtures instead:

https://docs.pytest.org/en/6.2.x/tmpdir.html

Reply:
got it, thanks!